docs/development/build-instructions-windows.md

```diff
@@ -41,6 +41,7 @@ there is no Visual Studio project generated.
 $ cd electron
 $ python script\bootstrap.py -v
 ```
+To avoid errors running the bootstrap script, run the bootstrap script as administrator
```

Comment:
I don't think this is correct; Electron doesn't require the bootstrap script to be run as administrator AFAIK.

What errors did you encounter that required admin permissions?

Reply:
Sorry, I don't remember the error but I need it to finalize the bootstrap. Also, I did try to clone electron in a different folder and now I am able to complete the bootstrap without administrator rights. My guess, at first time, the script tried to do something that requires running as administrator and now it is not required.